templates/components/datatable.html.twig

```diff
@@ -76,9 +76,9 @@
                     {% set super_header_label = super_header is array ? super_header['label'] : super_header %}
                     {% set super_header_raw = super_header is array ? super_header['is_raw'] : false %}
                     <tr>
-                        <th colspan="{{ total_cols }}">
+                        {% if super_header_raw != 'th_elements' %}<th colspan="{{ total_cols }}">{% endif %}
```

Comment:
It sound strange to provide th elements as raw html here.

Reply:
The alternative is refactoring the datatable template to allow a lot more customization for edge-cases.
I suppose it needs done eventually if we want to get rid of the HTMLTable* classes.